src/pages/ActiveProjectsView.jsx

```diff
@@ -1,5 +1,30 @@
 import React from 'react';
 
+import FeatureCopy from '../components/FeatureContainer/FeatureCopy';
+import Button from '../components/Objects/Button/Button';
+import SocialMediaConstants from '../constants/social-media-constants';
+import ActiveProjectsSVG from '../assets/ActiveProjectsSVG/ActiveProjectsSVG';
+import FeatureContainer from '../components/FeatureContainer/FeatureContainer';
+
+
 export default function ActiveProjectsView() {
-  return <h1>Active Projects</h1>;
+  /************************************
+   * Render
+   ************************************/
+
+  return (
+    <div className="active-projects-view-container">
+      <div className="active-projects-view-body">
+          <div className="active-projects-view-copy-container">
+            <h1>Active Projects</h1>
```

Comment:
Should that be just Projects instead?

Reply:
Should I change the navbar, too? or should that just be a separate ticket?